src/decrypt.rs

```diff
@@ -60,53 +58,38 @@ impl Decryptor {
     ///           - \<filename>:fd=\<filedescriptor> \
     ///           - \<filename>:\<password> \
     ///           - provider:<cmd/gprc>
-    pub async fn get_plaintext_layer(
+    pub fn get_plaintext_layer(
```

Comment:
Could you add a few unit tests for the two functions in this file (even if you just test a few of the error paths)?